src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx

```diff
@@ -0,0 +1,215 @@
+import { ContextModule, OwnerType } from "@artsy/cohesion"
+import {
+  Box,
+  Flex,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+  Text,
+} from "@artsy/palette"
+import { shuffle } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Rail } from "Components/Rail"
+import { useTracking } from "react-tracking"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { trackHelpers } from "Utils/cohesionHelpers"
+import { extractNodes } from "Utils/extractNodes"
+import { PreviouslySoldOnArtsyRailQuery } from "__generated__/PreviouslySoldOnArtsyRailQuery.graphql"
+import { PreviouslySoldOnArtsyRail_recentlySoldArtworks$data } from "__generated__/PreviouslySoldOnArtsyRail_recentlySoldArtworks.graphql"
+import { SoldArtworkFragmentContainer } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SoldArtwork"
+
+interface PreviouslySoldOnArtsyRailProps {
+  recentlySoldArtworks: PreviouslySoldOnArtsyRail_recentlySoldArtworks$data
+}
+
+export const PreviouslySoldOnArtsyRail: React.FC<PreviouslySoldOnArtsyRailProps> = ({
+  recentlySoldArtworks,
+}) => {
+  const tracking = useTracking()
+
+  if (!recentlySoldArtworks) {
+    return null
+  }
+
+  const artworks = shuffle(extractNodes(recentlySoldArtworks)).filter(
+    ({ artwork }) => !!artwork
+  )
+
+  const trackArtworkItemClick = (artwork, horizontalSlidePosition) => () => {
+    tracking.trackEvent(
+      trackHelpers.clickedArtworkGroup(
+        ContextModule.artworkRecentlySoldGrid,
+        OwnerType.consign,
+        artwork.internalID,
+        artwork.slug,
+        horizontalSlidePosition
+      )
+    )
+  }
+
+  return (
+    <Rail
+      title="Previously sold on Artsy"
+      showProgress={false}
+      getItems={() => {
+        return artworks.map(
+          ({ artwork, lowEstimate, highEstimate, priceRealized }, i) => {
+            if (!artwork) return <></>
+
+            return (
+              <SoldArtworkFragmentContainer
+                key={artwork.internalID}
+                artwork={artwork}
+                lazyLoad
+                data-testid="previouslySoldItem"
+                onClick={trackArtworkItemClick(artwork, i)}
+                // FIXME:
+                contextModule={ContextModule.artworkRecentlySoldGrid as any}
+              >
+                <Flex justifyContent="space-between" mb={[0.5, 1.5]}>
+                  <Text variant={["lg-display", "xl"]} overflowEllipsis>
+                    {priceRealized?.display}
+                  </Text>
+                  <Text
+                    variant={["lg-display", "xl"]}
+                    color="green100"
+                    overflowEllipsis
+                  >
+                    123%
```

Comment:
Where is this value from?

Reply:
Oops, i wanted to ask in the channel where to get the info from and didn't 👀 
wip